libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -42,20 +42,20 @@ extern const AP_HAL::HAL& hal;
 
 
 const AP_Param::GroupInfo AC_Fence::var_info[] = {
+
     // @Param: ENABLE
     // @DisplayName: Fence enable/disable
-    // @Description: Allows you to enable (1) or disable (0) the fence functionality
-    // @Values: 0:Disabled,1:Enabled
```

Comment:
need to re-add @Values

Reply:
and an accurate description...havent reviewed...but if ALTMIN is configured, does setting this enable it even if below the ALTMIN height? how that functions should be in the description also

Reply:
Fixed